src/github/client/github-client.ts

```diff
@@ -127,11 +127,11 @@ export class GitHubClient {
 		);
 		this.axios.interceptors.response.use(
 			instrumentRequest(metricHttpRequest.github, this.restApiUrl, jiraHost, {
-				withApiKey: "" + (!!gitHubConfig.apiKeyConfig),
+				withApiKey: (!!gitHubConfig.apiKeyConfig).toString(),
 				...this.metrics
 			}),
 			instrumentFailedRequest(metricHttpRequest.github, this.restApiUrl, jiraHost, {
-				withApiKey: "" + (!!gitHubConfig.apiKeyConfig),
+				withApiKey: (!!gitHubConfig.apiKeyConfig)?.toString() || "undefined",
```

Comment:
nit: pretty sure !! is always defined :) should've duplicated my prev. comment, feel free to ignore!